config.json

```diff
@@ -1,223 +1,271 @@
 {
   "language": "TypeScript",
   "active": true,
+  "blurb": "TypeScript is a language for application-scale JavaScript. TypeScript adds optional types, classes, and modules to JavaScript.",
   "test_pattern": ".*[.]test[.]ts$",
   "exercises": [
     {
       "uuid": "755f6f85-9ab3-4af6-9e35-dbb9d64be9c5",
       "slug": "hello-world",
-      "core": false,
-      "unlocked_by": null,
+      "core": true,
       "difficulty": 1,
       "topics": [
-        "strings"
+        "Control-flow (conditionals)",
+        "Optional values",
+        "Strings",
+        "Text formatting"
```

Comment:
This PR includes the changes from the other one. Would you mind tweaking this so it only contains the copy changes?